Cargo.toml

```diff
@@ -10,11 +10,11 @@ repository = "https://github.com/MagicalBitcoin/rust-hwi"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-bitcoin = { version = "0.25", features = ["use-serde", "base64"] }
-serde = { version = "^1.0", features = ["derive"] }
-serde_json = { version = "^1.0" }
-strum_macros = "0.18.0"
-base64 = "^0.11"
+bitcoin = { version = "0.28.1", features = ["use-serde", "base64"] }
+serde = { version = "1.0.137", features = ["derive"] }
+serde_json = { version = "1.0.81" }
```

Comment:
It's better to leave `^1.0` here (for both `serde` and `serde_json`), it basically means "whatever version >= 1.0 but < 2.0". This way we still have all the fixes they might push after 1.0.137. [BDK uses the same](https://github.com/bitcoindevkit/bdk/blob/master/Cargo.toml#L19-L20) :)